src/ml/test/s1.txt

```diff
@@ -0,0 +1,3 @@
+A B 1
```

Comment:
We can follow the example `summary/summary.py` uses for its test code in `test/analysis`. There is a file with test code and subdirectories for input and output files.